glean-core/csharp/Glean/Dispatchers.cs

```diff
@@ -95,39 +95,42 @@ internal static Task LaunchAPI(Action action)
         {
             Task task = null;
 
+            // Wrap the provided action in a try/catch block: we don't want to
+            // break execution if something throws.
+            Action safeAction = () => {
+                // In order to prevent tasks from causing exceptions
+                // we wrap the action invocation in try/catch
```

Comment:
A little comment redundancy here, I think you should pick one and delete the other.